test/integration/Mac/AlertTest.cs

```diff
@@ -0,0 +1,40 @@
+﻿using Appium.Net.Integration.Tests.helpers;
+using NUnit.Framework;
+using OpenQA.Selenium.Appium;
+using OpenQA.Selenium.Appium.Enums;
+using OpenQA.Selenium.Appium.Mac;
+
+namespace Appium.Net.Integration.Tests.Mac
+{
+    [Ignore("Ignore tests for macOS on CI")]
```

Comment:
This can be removed as there isn't a filter for Mac OS. When a filter will be added to CI, it'll grep by namespace